client/document/index.jsx

```diff
@@ -65,6 +66,7 @@ class Document extends React.Component {
 
 		const inlineScript =
 			`COMMIT_SHA = ${ jsonStringifyForHtml( commitSha ) };\n` +
+			`BUILD_TIMESTAMP = ${ jsonStringifyForHtml( buildTimestamp ) };\n` +
```

Comment:
both of these could use `var` before them. Not sure why we missed it with the COMMIT_SHA.

IE: `var COMMIT_SHA = `

Reply:
Done and done.